lib/http/request.ts

```diff
@@ -142,8 +158,74 @@ export function httpRequest(sdk: OktaAuthHttpInterface, options: RequestOptions)
     withCredentials
   };
 
-  var err, res;
-  return sdk.options.httpRequestClient!(method!, url!, ajaxOptions)
+  var err, res, promise;
+
+  if (isIOS()) {
```

Comment:
My initial research/assumptions about browser version detection (I seemed to have confused it with OS version) were wrong. We can actually determine the browser version. This is a library that does the parsing: https://www.npmjs.com/package/ua-parser-js

I think it's fine to use it or use it as a reference. Either way we should gate this by the specific browser versions `18.x`

Reply:
Done
https://github.com/okta/okta-auth-js/pull/1552/commits/793382cf356bd9f1c79965d1c9f3f3b31fe98391#diff-b640bdcca3985bce5a6f136070067d742a100754b211417c74125f85a9be47b2R100